python-project/test/test_jp.py

```diff
@@ -119,6 +121,4 @@ def ocr_test_case(test_case: OCRTest, index: int) -> None:
 
     print(f'{name} {index + 1}/{test_case.count} result: {result}')
 
-    # TODO compare against actual string reference data
-
-    assert False, "add reference string and compare here"
+    assert result == expected_text
```

Comment:
Removed the custom failure message so we get a nice summary of what went wrong for each test.